noetic-foxy/package.xml

```diff
@@ -18,61 +18,103 @@
+  <depend condition="$ROS_VERSION == 1">image_proc</depend>
+  <depend condition="$ROS_VERSION == 1">image_view2</depend>
+  <depend condition="$ROS_VERSION == 1">imu_filter_madgwick</depend>
+  <depend condition="$ROS_VERSION == 1">imu_tools</depend>
+  <depend condition="$ROS_VERSION == 1">joint_state_controller</depend>
+  <depend condition="$ROS_VERSION == 1">jsk_recognition_msgs</depend>
+  <depend condition="$ROS_VERSION == 1">jsk_recognition_utils</depend>
+  <depend condition="$ROS_VERSION == 1">jsk_rviz_plugins</depend>
+  <depend condition="$ROS_VERSION == 1">jsk_topic_tools</depend>
+  <depend condition="$ROS_VERSION == 1">nodelet</depend>
+  <depend condition="$ROS_VERSION == 1">position_controllers</depend>
+  <depend condition="$ROS_VERSION == 1">python_qt_binding</depend>
+  <depend condition="$ROS_VERSION == 1">rosbag</depend>
+  <depend condition="$ROS_VERSION == 1">rosbag_storage</depend>
+  <depend condition="$ROS_VERSION == 1">rosconsole</depend>
+  <depend condition="$ROS_VERSION == 1">roscpp</depend>
+  <depend condition="$ROS_VERSION == 1">roscpp_serialization</depend>
+  <depend condition="$ROS_VERSION == 1">roslaunch</depend>
+  <depend condition="$ROS_VERSION == 1">roslib</depend>
+  <depend condition="$ROS_VERSION == 1">roslint</depend>
+  <depend condition="$ROS_VERSION == 1">rospy</depend>
+  <depend condition="$ROS_VERSION == 1">rostest</depend>
+  <depend condition="$ROS_VERSION == 1">rosunit</depend>
+  <depend condition="$ROS_VERSION == 1">rqt_gui</depend>
+  <depend condition="$ROS_VERSION == 1">rqt_gui_py</depend>
+  <depend condition="$ROS_VERSION == 1">rqt_plot</depend>
+  <depend condition="$ROS_VERSION == 1">rviz</depend>
+  <depend condition="$ROS_VERSION == 1">socketcan_bridge</depend>
+  <depend condition="$ROS_VERSION == 1">socketcan_interface</depend>
+  <depend condition="$ROS_VERSION == 1">sound_play</depend>
+  <depend condition="$ROS_VERSION == 1">swri_math_util</depend>
+  <depend condition="$ROS_VERSION == 1">swri_roscpp</depend>
+  <depend condition="$ROS_VERSION == 1">swri_serial_util</depend>
+  <depend condition="$ROS_VERSION == 1">swri_string_util</depend>
+  <depend condition="$ROS_VERSION == 1">swri_nodelet</depend>
+  <depend condition="$ROS_VERSION == 1">tablet_socket_msgs</depend>
+  <depend condition="$ROS_VERSION == 1">tf</depend>
+  <depend condition="$ROS_VERSION == 1">tf_conversions</depend>
+  <depend condition="$ROS_VERSION == 1">topic_tools</depend>
+  <depend condition="$ROS_VERSION == 1">transmission_interface</depend>
+  <depend condition="$ROS_VERSION == 1">vector_map_msgs</depend>
+  <depend condition="$ROS_VERSION == 1">velodyne_description</depend>
+  <depend condition="$ROS_VERSION == 1">velodyne_gazebo_plugins</depend>
+  <depend condition="$ROS_VERSION == 1">velodyne_pcl</depend>
+  <depend condition="$ROS_VERSION == 1">xmlrpcpp</depend>
+
+  <!-- ROS 2 specific dependencies -->
+  <depend condition="$ROS_VERSION == 2">rclcpp</depend>
```

Comment:
Looks like these packages are also available as debian distibutions, lets install them through apt like the other foxy packages

Reply:
removed the 2 ROS2 package (lifecycle is already included in ros-foxy-desktop)
however, rclcpp says just `minimal` ones, which I wasnt sure if it was exactly same, so I made sure to include it as ros-foxy-rclcpp